README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
```

Comment:
I think this whole thing should be in its own file that should be linked to in the general repo (I think we talked about this @paulmiard last time, maybe I'm wrong). My general rationale is that, with this being open source, it doesn't make sense to talk about file structure -- we're not mandating a project file structure for our style guide as much as a guideline for our internal product teams.